components/toolbar/appearance.md

```diff
@@ -12,6 +12,41 @@ position: 35
 
 This article outlines the available ToolBar parameters, which control its appearance.
 
+## FillMode
+
+The `FillMode` parameter controls if the ToolBar will have a background and borders. To set the parameter value, use the `string` members of the static class `ThemeConstants.ToolBar.FillMode`.
+
+| `FillMode` Class Member | String Value |
```

Comment:
Would it possible to add a third column that describes the corresponding behavior?

Reply:
It is not our practice to do so. See other articles about the same feature:
https://docs.telerik.com/blazor-ui/components/button/appearance#fillmode
https://docs.telerik.com/blazor-ui/components/dropdownbutton/appearance#fillmode
https://docs.telerik.com/blazor-ui/components/numerictextbox/appearance#fillmode
Additionally, the example showcases all the possibilities and it's visually shown to the reader.